.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+## Description
+
+TODO: Include a detailed description of the changes in the body of the PR.
+
+TODO: if relevant, provide a URL to updated Pattern Library documentation.
+
+## Manual Test Plan
+
+TODO: Include instructions for any required manual tests, and any manual testing that has
+already been performed.
+
+## Testing Checklist
+- TODO: Include unit tests
+- TODO: Include accessibility (a11y) tests
+- [ ] Manually test right-to-left languages and i18n
+  of the changes.
+- [ ] Manually test responsive behavior.
+
+## Non-testing Checklist
+- TODO: Tag DevOps on your PR (user `edx/devops`), either for review
+  (more specific to devops = better) or just a general FYI.
+- TODO: Consider any documentation your change might need, and which
+  users will be affected by this change.
+- TODO: Double-check that your commit messages will make a meaningful release note.
```

Comment:
I agree, I'm taking this line out and capturing the spirit of it in the post-test checklist with a note to make sure squashed commits have descriptive messages. (FYI I'm taking this PR over from Andy.)